### ISSUE TITLE
ACF plot blank for some surfaces

### ISSUE THREAD

**Issue by lthimons** [ComputationalMechanics/TopoBank#577]
When I try to analyze ACF for microcrystalline diamond and nanocrystalline diamond (shared with me by Abhijeet) the ACF plot is blank. I can select a subset of topographies from the TEM (.asc files) and the AFM (.001,.002,etc.) and calculate the ACF. When any of the stylus (.txt) data is selected, the plot shown is empty.

**Comment by lthimons** [comment]
![ncd_acf](https://user-images.githubusercontent.com/50414930/108072332-c1fb1500-7034-11eb-9751-48722145beca.png)

There seems to be some issue with the auto-axis scaling. When I select ONLY the AFM and TEM data I can see data on the ACF plot, however it seems like it might be stick on the default axis bounds between 10^-1 and 10^1.

**Comment by mcrot** [comment]
Thank you for the bug report, I'll have a look at this.

**Comment by mcrot** [comment]
@lthimons , could you share the data with me inside the web app?

**Comment by lthimons** [comment]
The issue appears with the surface "Microcrystalline Diamond". I cannot share it as it was shared with me and I do not have the proper permission. However it looks like it was also shared with you. Do you have a surface with that name that contains TEM, AFM, and stylus data?

-Luke
________________________________

**Comment by mcrot** [comment]
Yes, I already have it, thanks. It was also published: https://contact.engineering/go/mz7z5/

**Comment by lthimons** [comment]
excellent. I just checked on my computer and was able to reproduce the issue. If you cannot reproduce it then I am happy to provide more details on my setup.

-Luke
________________________________
